golint/golint.go

```diff
@@ -28,8 +28,8 @@ var (
 func usage() {
 	fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
 	fmt.Fprintf(os.Stderr, "\tgolint [flags] # runs on package in current directory\n")
-	fmt.Fprintf(os.Stderr, "\tgolint [flags] package\n")
-	fmt.Fprintf(os.Stderr, "\tgolint [flags] directory\n")
+	fmt.Fprintf(os.Stderr, "\tgolint [flags] packages\n")
+	fmt.Fprintf(os.Stderr, "\tgolint [flags] directory...\n")
```

Comment:
Perhaps change all three lines to be:
```"\tgolint [flags] [packages]\n"```
```"\tgolint [flags] [directories] # where a '/...' suffix includes all sub-directories\n"```
```"\tgolint [flags] [files] # where all files belong to the same package\n"```

The use of "..." in the help is confusing since it is different from the ... expansion in the logic below. This matches the help message go vet better.